python/ml4ir/applications/classification/model/classification_model.py

```diff
@@ -232,7 +232,11 @@ def predict(
         # than a list on numpy arrays
         predictions_df[self.output_name] = [x for x in predictions_]
         if logs_dir:
-            np.set_printoptions(threshold=sys.maxsize)  # write the full vector in the csv not ...
+            np.set_printoptions(formatter={'all':lambda x: str(x.decode('utf-8')) if isinstance(x, bytes) else str(x)},
```

Comment:
did we check that this doesn't break @balikasg and @Ullimague 's evaluation spark job for entity prediction, when this CSV changes it's shape a little?

Reply:
I am aware of the change, I will adapt the spark job by simplifying it.